README.md

```diff
@@ -170,7 +170,38 @@ We have implemented a number of commands in the `nf-core/tools` package to make
 2. Install [`nextflow`](https://nf-co.re/usage/installation) (`>=20.11.0-edge`; see [Nextflow edge releases](#nextflow-edge-releases))
 3. Install any of [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) or [`Conda`](https://conda.io/miniconda.html)
 4. [Fork and clone this repo locally](#uploading-to-nf-coremodules)
-5. Create a module using the [nf-core DSL2 module template](https://github.com/nf-core/tools/blob/master/nf_core/module-template/software/main.nf):
+5. Set up git by adding a new remote of the nf-core git repo called `upstream`
+
+   ```bash
+   git remote add upstream https://github.com/nf-core/modules.git
+   ```
+
+   Make a new branch for your module and check it out
+
+   ```bash
+   git checkout -b fastqc
+   ```
+
+   On a regular basis you can pull upstream changes into this branch and it is recommended to do so before pushing and creating a pull request (PR) - see below. Rather than merging changes directly from upstream the rebase strategy is recommended so that your changes are applied on top of the latest master branch from the nf-core repo. This can be performed as follows
+
+   ```bash
+   git pull --rebase upstream master
+   ```
+
+   Once the PR has been accepted you should delete the branch and checkout master again.
+
+   ```bash
+   git checkout master
+   git branch -d fastqc
+   ```
+
+   In case there are commits on the local branch that didn't make it into the PR (usually commits made after the PR), git will warn about this and not delete the branch. If you are sure you want to delete, use the following command
```

Comment:
For modules, this will _always_ be the case though. Because we are using squash merges, so the commits made in the feature branch will never be merged into `master`. Only the single squash merge commit.

Reply:
I don't think this will impact deletion of the local branch with `-d`?